content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: 'setTimeout()'
+Description: 'Allows users to execute callbacks after a period of time expressed in milliseconds.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`setTimeout()`** is a function serviced globally by the window object provided by the user's browser.
+
+## Syntax
+
+`setTimeout()` is capable of receiving multiple parameters where the first is a callback function, the second parameter receives a number that represents the time in milliseconds (1s = 1000ms), that defines the time needed for the callback to execute, the third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
+
+The provided sample code contains some examples of how `setTimeout()` is composed:
+
+```pseudo
+setTimeout(callback)
+
+setTimeout(callback, delay)
+
+setTimeout(callback, delay, param)
+```
+
+## Example 1
+
+The following piece of code will shows an example of the `setTimeout()` implementation:
+
+```shell
+// Defining the function
+function sayHello() {
+  console.log('Hello from Codecademy');
+}
+
+// Calling the function using setTimeout.
+setTimeout(sayHello, 4000);
+```
+
+The output will be displayed after 4 seconds:
+
+```output
+Hello from Codecademy
+```
+
+## Example 2
+
+The following example shows the use of [arrow functions](https://www.codecademy.com/resources/docs/javascript/arrow-functions). Instead of declaring it using the word `function`, we can just create it by using `() =>`.
+
+```shell
+// Defining the function
+setTimeout(() => {
+  console.log('Hello World from Codecademy');
+}, 2000);
+```
+
+The output from the arrow function will be displayed after 2 seconds:
+
+```shell
+Hello World from Codecademy
+```
+
+## Codebyte Example
+
+[Codebyte](https://help.codecademy.com/hc/en-us/articles/1260806285369-Codebytes) is way to easily write, share, and edit executable code blocks directly in the Codecademy.
```

Comment:
```suggestion
```
You misunderstand. We don't need an explanation of what a codebyte is, we want an explanation of what the code in it does.